at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb

```diff
@@ -0,0 +1,53 @@
+class StringFormat
+  class << self
+  def human_name(string)
+    if string.is_a? String
+      i = 0
+      out = ''
+      string.strip!
+      string.gsub!(/[^A-Za-z ]/, '')
+      string.each_char do |x|
+        if string[i - 1] == ' ' && x != ' ' || i.zero?
+          _a = x.ord - 32
+          out += _a.chr
+        else
+          out += x unless(string[i - 1] == ' ' && x == ' ')
+        end
+        i += 1
+      end
+      out
+    else
+      nil
+    end
+  end
+
+  def uniq(string)
+    hash = {}
+    string.each_char { |_str| hash[_str] = ((hash[_str] || 0) + 1) }
+    hash.keys.inject { |out, i| out + i }
+  end
+
+  def only_letter?(string)
+    if string.is_a? String
+      string.strip!
+      i = 0
+      check = true
+      p string
+      string.each_char do |x|
+        if string[i - 1] == ' ' && x != ' '
+          check = false
+        end
+        i +=1
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/c08836bee3655e104837529ea8ff45ce098ad76f/at-luantran/Week3/2018_07_05_exercise4_rubyday3.rb#L40"></span>Layout/SpaceAroundOperators: Surrounding space missing for operator <code>+=</code>. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>